File: app/src/main/java/com/tes/aop/di/AppModule.java
package com.tes.aop.di;

import dagger.Module;
import dagger.hilt.InstallIn;
import dagger.hilt.components.SingletonComponent;

/**
 * AppModule — Hilt dependency injection module.
 *
 * TODO: Add @Provides methods here to inject dependencies.
 * Example: provide Retrofit, OkHttpClient, Repository, etc.
 *
 * @Provides
 * @Singleton
 * public Retrofit provideRetrofit() {
 *     return new Retrofit.Builder()
 *         .baseUrl(ApiClient.BASE_URL)
 *         .addConverterFactory(GsonConverterFactory.create())
 *         .build();
 * }
 */
@Module
@InstallIn(SingletonComponent.class)
public class AppModule {
    // Add @Provides methods here
}

File: app/src/main/java/com/tes/kk/di/AppModule.kt
package com.tes.kk.di

import dagger.Module
import dagger.hilt.InstallIn
import dagger.hilt.components.SingletonComponent

/**
 * AppModule — Hilt dependency injection module.
 *
 * TODO: Add @Provides or @Binds functions here.
 * Example:
 *
 * @Provides @Singleton
 * fun provideRetrofit(): Retrofit = Retrofit.Builder()
 *     .baseUrl(ApiClient.BASE_URL)
 *     .addConverterFactory(GsonConverterFactory.create())
 *     .build()
 */
@Module
@InstallIn(SingletonComponent::class)
object AppModule {
    // Add @Provides functions here
}
